apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx

```diff
@@ -15,15 +14,16 @@ import { ToggleOption } from '.'
 const ToggleOptionStory: Meta<typeof ToggleOption> = {
   ...journeysAdminConfig,
   component: ToggleOption,
-  title: 'Journeys-Admin/Editor/ControlPanel/Attributes/ToggleOption'
+  title:
+    'Journeys-Admin/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption'
 }
 
 const Template: StoryObj<typeof ToggleOption> = {
```

Comment:
`import { ComponentProps } from 'react'`
```suggestion
const Template: StoryObj<ComponentProps<typeof ToggleOption>> = {
```